app/queries/keywords_query.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class KeywordsQuery
+  def initialize(filters = {})
+    # NOTE(Goose97): we don't support any filters for now
+    @filters = filters
```

Comment:
If it is not needed, then let's remove it 😅 

```suggestion
def initialize()
  #TODO: add filters object as parameter
end
```

Prefer `TODO` over `# NOTE(Goose97):` as it is a standard format usually handled well in text editors (and IDEs).

Reply:
Fixed in d93acb0abbf3765eceda45d0cc3316b982af5365